keys.ts

```diff
@@ -0,0 +1 @@
+export const LIMIT_DISCLOSURE_SIGNATURES = ["RsaSignature2018", "Ed25519Signature2020"];
```

Comment:
These suites don't allow selective disclosure. The bbs+ suite is the only one

Reply:
Probably also wise to make the const name more explicit as it refers to signature suites and not signatures itself

Reply:
Fixed!